### PR TITLE
Yeehaw Ian

### LINKED ISSUE #27054: Ian Can't Wear Cowboy Hats
(WEB REPORT BY: rubylips REMOTE: 206.221.180.138:7777)
# Description
Ian can't wear cowboy hats.

# Steps to Reproduce
Try to put a cowboy hat on Ian

# What you Expected
Yeehaw Ian

# What Actually Happened
Ian

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #27054 

image

:cl:
* rscadd: Ian can now wear cowboy hats.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/corgi.dm [hunk 1]
```diff
@@ -464,6 +464,12 @@
 			else
 				to_chat(usr, "<span class = 'notice'>The glow of /the [V] startles [real_name]!</span>")
 
+		if(/obj/item/clothing/head/cowboy)
+			name = "Yeehaw Ian"
+			desc = "Are you really just gonna stroll past without saying howdy?"
```
Comment (boy2mantwicethefam) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

Comment (Kharkov72) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

Comment (drawsstuff) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

**Comment by Kammerjunk** [comment]
🤠

File: code/modules/mob/living/simple_animal/friendly/corgi.dm (hunk 1 above)
Comment (beesil) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

Comment (PrimeDSS13) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

Comment (Blithering) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: Howdy

**Comment by tuypo1** [comment]
Well I'm not gonna just stroll on by without saying howdy

File: code/modules/mob/living/simple_animal/friendly/corgi.dm (hunk 1 above)
Comment (kilozombie) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

Comment (SonixApache) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

Comment (GreatBigFailure) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

**Comment by GreatBigFailure** [comment]
Yeehaw

File: code/modules/mob/living/simple_animal/friendly/corgi.dm (hunk 1 above)
Comment (cloutcode) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

Comment (AI-Is-Rogue) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:469]: howdy

**Pull request merged**